### PR TITLE
Improvements to scenario & regression tests

### PR DESCRIPTION
**Pull request by Psirex**

### CODE REVIEW

File: test/regressions/dg-solvency-simulation.t.sol [hunk 1]
```diff
@@ -0,0 +1,1656 @@
...
+
+            // console.log("Account %s submitted %s stETH.", account, submitAmount.formatEther(), balance.formatEther());
+        }
+    }
+
+    function _submitWstETHByRandomAccount(address[] storage accounts) internal {
+        // console.log(">>> Submitting wstETH by random account");
+        uint256 randomIndexOffset = _random.nextUint256(accounts.length);
+
+        for (uint256 i = 0; i < accounts.length; ++i) {
```
Comment (Psirex) [inline test/regressions/dg-solvency-simulation.t.sol:929]: Same : "Here happens submission by all simulation accounts, but it is expected to be only one random account"

File: README.md [hunk 2]
```diff
@@ -36,11 +47,68 @@ This project uses NPM for dependency management and Forge for tests so you'll ne
...
+    - **Scenario tests** - Usually edge cases that demonstrate how the system behaves under very specific conditions. These tests use fresh deployments of the DG contracts (and forked Lido contract state) to prepare the system state for concrete scenarios.
+    - **Regression tests** - Integration tests that can be launched on forked instances of the DG contracts. These tests verify the required functionality of the system and ensure everything works exactly as expected. This category contains two special tests: 
+        1) A [test for complete rage quit](./test/regressions/complete-rage-quit.t.sol) of the majority of real stETH and wstETH holders. 
+        2) A special [solvency test](./test/regressions/dg-solvency-simulation.t.sol) that simulates protocol operation under conditions with serial Rage Quits. 
+    
+            Both tests may require significant time to complete, so they are not expected to be run frequently, in contrast to regular regression tests which can be run daily to verify the system works correctly.
+
+The following commands can be used to run different types of tests:
+
+- **Run unit tests exclusively**
```
Comment (bulbozaur) [inline README.md:67]: List of required env variables is necessary to be listed here

File: test/regressions/regression-test-utils/download_vetoers.js [hunk 3]
```diff
@@ -1,47 +1,76 @@
...
+  __dirname,
+  "..",
+  "complete-rage-quit-files",
+  `${FILENAME_PREFIX}steth_vetoers.json`
+);
 const ST_ETH_HOLDERS_EXCLUDE_ADDRESSES = new Set([
-    "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0", // WstETH
-    "0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1" // WithdrawalQueue
+  "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0".toLowerCase(), // WstETH
+  "0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1".toLowerCase(), // WithdrawalQueue
```
Comment (bulbozaur) [inline test/regressions/regression-test-utils/download_vetoers.js:20]: Maybe good add more Lido contracts that could contain steth

File: README.md (hunk 2 above)
Comment (Psirex) [reply README.md:67]: Fully agree! Propose adding this via a standalone PR to avoid increasing the scope of the current one.

File: test/regressions/regression-test-utils/download_vetoers.js (hunk 3 above)
Comment (Psirex) [reply test/regressions/regression-test-utils/download_vetoers.js:20]: Agreed, we should research which contracts hold stETH (e.g., InsuranceFund, Burner, Staking Modules, etc.). Let's add it in a standalone PR.
